internal/controller/streams_helper.go

```diff
@@ -29,6 +29,21 @@ func sendObjectData(sm *streamManager, metadata *pb.KubernetesObjectData) error
 	return sendToResourceStream(sm.logger, sm.streamClient.resourceStream, request)
 }
 
+// sendFalcoFlow sends a FalcoFlow to the networkFlowsStream
+func sendFalcoFlow(sm *streamManager, flow *pb.FalcoFlow) error {
```

Comment:
Maybe if possible make a generalized flows send function since we are using the same stream for all flows. This might be helpful with all of the future cni plugins we will use and not have a lot fo duplicate code. Thanks!